app/src/main/java/dev/msfjarvis/lobsters/ui/viewmodel/LobstersViewModel.kt

```diff
@@ -5,48 +5,51 @@ import androidx.lifecycle.viewModelScope
 import androidx.paging.Pager
 import androidx.paging.PagingConfig
 import dagger.hilt.android.lifecycle.HiltViewModel
+import dev.msfjarvis.lobsters.data.local.LobstersPost
 import dev.msfjarvis.lobsters.data.remote.LobstersPagingSource
-import dev.msfjarvis.lobsters.data.source.PostsDatabase
-import dev.msfjarvis.lobsters.model.LobstersPost
+import dev.msfjarvis.lobsters.data.repo.LobstersRepository
 import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class LobstersViewModel @Inject constructor(
   private val pagingSource: LobstersPagingSource,
-  database: PostsDatabase,
+  private val lobstersRepository: LobstersRepository,
 ) : ViewModel() {
   private val _savedPosts = MutableStateFlow<List<LobstersPost>>(emptyList())
-  private val savedPostsDao = database.savedPostsDao()
-  val savedPosts: StateFlow<List<LobstersPost>> get() = _savedPosts
-  val posts = Pager(PagingConfig(25)) {
+  val savedPosts = _savedPosts.asStateFlow()
+  val posts = Pager(PagingConfig(5)) {
```

Comment:
Why this change? This is what loads the data from the remote server, and the page size is 25 in the API.

Reply:
Yeah, this was a test change that I did to try and fetch lesser number of posts for debugging. This should not have been a part of the commit. I'll revert it along with the other changes that are required.